__tests__/push/bundler.test.ts

```diff
@@ -55,4 +55,16 @@ describe('Bundler', () => {
     const content = await bundler.build(journeyFile, generateTempPath());
     await validateZip(content);
   });
+
+  it('throw errors on incorrect path', async () => {
+    const bundler = new Bundler();
+    try {
+      await bundler.build(
+        join(dirname(journeyFile), 'blah.ts'),
+        generateTempPath()
+      );
+    } catch (e) {
+      expect(e.message).toContain('ENOENT');
```

Comment:
NIT: I was thinking we could maybe add a more informative error message when we pick-up any non-existing files?

Reply:
These come from ESbuild directly, not sure its worth changing it around.